src/utils/packet.js

```diff
@@ -0,0 +1,37 @@
+'use strict'
+
+const IlpPacket = require('ilp-packet')
+const assert = require('assert')
+const { omitUndefined } = require('../utils')
+const base64url = require('../utils/base64url')
+const debug = require('debug')('ilp:packet')
+
+const serialize = IlpPacket.serializeIlpPayment
+const parse = (packet) => {
+  try {
+    return IlpPacket.deserializeIlpPayment(packet)
+  } catch (e) {
+    debug('error while parsing packet: ' + e.message)
+    return undefined
+  }
+}
+  
+function getFromTransfer (transfer) {
+  assert(transfer, 'transfer must be defined. got: ' + transfer)
+  assert(typeof transfer === 'object', 'got invalid transfer: ' + transfer)
+  assert(Buffer.isBuffer(transfer.ilp || transfer.data),
+    'transfer.ilp or transfer.data must be buffer')
+
+  return transfer.ilp || transfer.data
```

Comment:
I don't see the point in possibly returning `transfer.data`. We're not actually making this change backwards compatible and the updated spec says the ILP packet should be under the `ilp` key.

Having this small wrapper around the `ilp-packet` module doesn't make that much sense to me. Why not just use the `ilp-packet` class directly where these functions are being used?